src/window/mod.rs

```diff
@@ -264,6 +263,7 @@ pub fn main_loop(
         save_window_size(
             window.is_maximized(),
             window.inner_size(),
+            window_wrapper.get_grid_size(),
```

Comment:
We should really fix the CI to properly run, but you need to add this to the linux version below as well.

Reply:
Looks like I missed another conditional compilation, sorry about that! I'll fix that.